packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -481,7 +481,7 @@ class RetainingPathWidget extends StatelessWidget {
   }
 }
 
-String? _containerClass(ObjRef? object) {
+String? _instanceClass(ObjRef? object) {
```

Comment:
nit: _instanceClassName to indicate that it returns a string and not a Class

Reply:
Done